detekt-gradle-plugin/src/main/kotlin/io/gitlab/arturbosch/detekt/Detekt.kt

```diff
@@ -228,7 +237,8 @@ abstract class Detekt @Inject constructor(
             AllRulesArgument(allRulesProp.getOrElse(false)),
             AutoCorrectArgument(autoCorrectProp.getOrElse(false)),
             BasePathArgument(basePathProp.orNull),
-            DisableDefaultRuleSetArgument(disableDefaultRuleSetsProp.getOrElse(false))
+            DisableDefaultRuleSetArgument(disableDefaultRuleSetsProp.getOrElse(false)),
+            RunRuleArgument(ruleToRun + ""),
```

Comment:
```suggestion
            RunRuleArgument(ruleToRun),
```

`ruleToRun` is a `String` already. You don't need ` + ""`, right?

Reply:
Indeed.